### ISSUE TITLE
writeHtml.py outputs difference slices across subjects

### ISSUE THREAD

**Issue by zhangfanmark** [pnlbwh/TBSS#29]
Hi @tashrifbillah 

Recently, Ofer @oferpasternak found out that the output images for the tbss qc script are not identical across the different subjects. The ideal way is have the same slices shown because one would expect the skeleton should be the same from the corresponding slices across subjects.

Kevin @kcho has made such changes at https://github.com/kcho/TBSS/blob/master/lib/writeHtml.py.

I tested his changes and worked well for me. I think he will do a pull request and ask you to review if it is reasonable to you.

But one issue I found out is that the images still a little different (see the arrows for two subjects). This might be due to an issue when saving the images into PNG, there should some interpolation here. 

![Screen Shot 2019-11-14 at 9 48 25 AM](https://user-images.githubusercontent.com/7855446/68869793-bef1d600-06c7-11ea-8ced-c7c4aeb0bcb4.png)

**Comment by tashrifbillah** [comment]
`writeHtml.py` leverages upon https://nilearn.github.io/modules/generated/nilearn.plotting.plot_stat_map.html at 
https://github.com/pnlbwh/TBSS/blob/master/lib/writeHtml.py#L92

So, if slices from same position across images are not being printed, it should be an issue with the above `nilearn` package.

> Recently, Ofer @oferpasternak found out that the output images for the tbss qc script are not identical across the different subjects.

@zhangfanmark I don't know why images from different subjects would be identical.

> This might be due to an issue when saving the images into PNG, there should some interpolation here.

Can you rephrase what you are trying to say?

If @oferpasernak thinks https://github.com/kcho/TBSS/blob/master/lib/writeHtml.py#L122 is the right way to plot, feel free to submit a PR.

**Comment by kcho** [comment]
@zhangfanmark Would you mind try running commit 2668aa590d50100b6b9495d23e839f7885b510c4 to see whether 'nearest' fixes the problem that you had, please?

**Comment by tashrifbillah** [comment]
PR https://github.com/pnlbwh/TBSS/pull/30

Status: 
@zhangfanmark should clarify the issue.
@kcho should respond the @tashrifbillah 's comments and suggestion on his PR.

**Comment by zhangfanmark** [comment]
> @zhangfanmark I don't know why images from different subjects would be identical.
> 
> > This might be due to an issue when saving the images into PNG, there should some interpolation here.

Sorry for the confusion. I meant the skeleton should be identical across subjects, at same location. So, if we have binary maps for the above two subjects. They should be the same.

**Comment by tashrifbillah** [comment]
@tashrifbillah 's design:

* have one cut_coords for enigma templates, (--enigma flag)
* have one cut coords for fmrib templates (--study flag)
* use the default settings for study templates (nothing is given)
* also accept cut_coords from the user, override any pre-specified cut_coords if one is provided

Assignment:
@kcho will give us --enigma and --fmrib cut_coords while @tashrifbillah will integrate the parameters in `writeHtml.py` acknowledging @kcho

**Comment by kcho** [comment]
@tashrifbillah here you go.

To document, these `cut_coords` provided below are not meant to highlight specific structures. It is given in order to keep the output slice in `writeHtml.py` constant for all subjects.

# `cut_coords` FMRIB
**-17, -24, 14**

![image](https://user-images.githubusercontent.com/3075783/69480046-eb95a400-0dd1-11ea-816b-9b3f009da63f.png)


# `cut_coords` ENIGMA
**1, -19, 14**

**Comment by tashrifbillah** [comment]
Fixed by PR https://github.com/pnlbwh/TBSS/pull/32
